app/main.development.js

```diff
@@ -179,6 +181,14 @@ const launchDCRWallet = () => {
     args.push('--testnet');
   }
 
+  // Add any extra args if defined.
+  if (argv.extrawalletargs != undefined) {
+    var extraArgs = argv.extrawalletargs.split(' ');
```

Comment:
So this would be very easy in a language with a real regex library (PCRE).  Unfortunately, js regex doesn't suppoer negative lookbehing which is the easiest way to catch that case.

How would you feel about this going in as is and them I open a new issue to allow extraargs with spaces (that way I can work on the things that we wanted this for and then worry about the less likely case later)/